### PR TITLE
Ember 3.24

### PR DESCRIPTION
**Pull request by svogt0511**
## Purpose

closes: #735 

Updates Fabrica to Ember 3.24, Node 18.  See the [epic](https://github.com/datacite/bracco/issues/735) for more details.

Manual testing checklist is [here](https://docs.google.com/document/d/18wIQyKQbiX20cOvsZclVya0CB3uGH0AUMWsuQ3fbZrs/edit).

VERCEL DEPLOYMENT: https://bracco-3c632crit-datacite.vercel.app/

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

- [ ] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Reviewer, please remember our [guidelines](https://datacite.atlassian.net/wiki/spaces/TEC/pages/1168375809/Pull+Request+Guidelines):

- Be humble in the language and feedback you give, ask don't tell.
- Consider using positive language as opposed to neutral when offering feedback. This is to avoid the negative bias that can occur with neutral language appearing negative.
- Offer suggestions on how to improve code e.g. simplification or expanding clarity.
- Ensure you give reasons for the changes you are proposing.

### CODE REVIEW

File: .travis.yml
Comment (jrhoads) [inline .travis.yml:1]: Is this still testing on travis?

File: cypress/e2e/organization_admin/repositories.test.ts [hunk 1]
```diff
@@ -80,21 +103,22 @@ describe('ACCEPTANCE: ORGANIZATION_ADMIN | REPOSITORIES', () => {
...
-      cy.get('a#account_menu_link').should('contain', 'DATACITE');
+      cy.get('a#account_menu_link').should('contain', Cypress.env('organization_admin_username').toUpperCase());
 
       cy.get('h3.edit').contains(/Add\s*Repository/);
 
       cy.get('#repository-id').should('be.visible');
       cy.get('#client-type').should('be.visible');
       cy.get('#client-type .ember-power-select-selected-item').should('contain', 'Repository');
 
+      /* - TEMPORARILY SKIP THIS
```
Comment (jrhoads) [inline cypress/e2e/organization_admin/repositories.test.ts:119]: How temporary.  Is there a timeline to revisit this?

File: cypress/e2e/staff_admin/prefix.test.ts [hunk 2]
```diff
@@ -8,11 +8,17 @@ describe('Admin: Prefix', () => {
   before(function () {
     cy.login(Cypress.env('staff_admin_username'), Cypress.env('staff_admin_password'));
     cy.setCookie('_consent', 'true');
+    cy.wait(waitTime2);
   })
 
   beforeEach(() => {
-    Cypress.Cookies.preserveOnce('_fabrica', '_jwt', '_consent');
-    cy.wait(waitTime2);
+    // TBD - set up test environment
```
Comment (jrhoads) [inline cypress/e2e/staff_admin/prefix.test.ts:15]: I've seen this a couple of times.   Is this still TBD in all of these cases?  Cypress no longer needs the cookies preserved?

File: package.json [hunk 3]
```diff
@@ -15,36 +15,35 @@
   },
   "scripts": {
     "build": "NODE_OPTIONS=--openssl-legacy-provider ember build --environment=production",
-    "lint": "./node_modules/.bin/eslint . --ext .js",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'",
```
Comment (jrhoads) [inline package.json:18]: Cool

**Review by jrhoads — APPROVED** [review]
Great work everyone!  This was a lot of work.

I have comments with some questions or observations in them, but nothing blocking.

File: .travis.yml
Comment (svogt0511) [reply .travis.yml:1]: No use of travis at all.  Will get rid of this.  I needed a better checklist to not forget these things.

File: cypress/e2e/staff_admin/prefix.test.ts (hunk 2 above)
Comment (svogt0511) [reply cypress/e2e/staff_admin/prefix.test.ts:15]: That should be gone.  It should not be used at all.  Will fix it and re-run all tests.

File: cypress/e2e/organization_admin/repositories.test.ts (hunk 1 above)
Comment (svogt0511) [reply cypress/e2e/organization_admin/repositories.test.ts:119]: Yes.  The timeline is the next Fabrica upgrade to Ember 3.28.  I left a provision under the heading of 'Improving Cypress Testing' to address the flakiness in tests that fill out forms.

**Pull request merged**